packages/genesys-spark-components/src/components/beta/gux-time-zone-picker/gux-time-zone-picker.tsx

```diff
@@ -76,11 +76,15 @@ export class GuxTimeZonePickerBeta {
   }
 
   private filterTimeZoneList(timeZoneList: GuxTimeZoneOption[]) {
-    const searchString = this.searchString;
+    const searchString = this.searchString.toLowerCase();
     return timeZoneList.filter(tzOption => {
-      return this.getFormattedTimeZoneOption(tzOption)
-        .toLowerCase()
-        .includes(searchString.toLowerCase());
+      return tzOption.mainCities.some(city => {
+        return tzOption.displayTextNameFormatted
```

Comment:
I feel like `displayTextNameFormatted` is a little confusing because it makes me wonder how exactly it's formatted and then I have to check elsewhere in this file to see. Perhaps we just format it when defining`displayTextName` initially and then just use that variable? Thoughts?

Reply:
I agree with this I dont think the names convey pretty well these were what they were called in the initial component implementation but I think they should change for sure 👍